index.d.ts

```diff
@@ -16,77 +18,34 @@ export type NodemonEventListener = {
   on(event: 'stdout' | 'stderr', listener: (e: string) => void): Nodemon;
   on(event: 'restart', listener: (e?: NodemonEventRestart) => void): Nodemon;
   on(event: 'quit', listener: (e?: NodemonEventQuit) => void): Nodemon;
-  on(event: 'exit', listener: (e?: NodemonEventExit) => void): Nodemon;
-  on(
-    event: 'config:update',
-    listener: (e?: NodemonEventConfig) => void
-  ): Nodemon;
+  on(event: 'exit', listener: (e?: number) => void): Nodemon;
```

Comment:
Why the NodemonEventExit is changed to number

Reply:
Check line 100, `export type NodemonEventExit = number;`. It was pretty useless to define this type imho.